ios/FluentUI/Table View/ListVnext.swift

```diff
@@ -214,6 +215,7 @@ extension MSFListView {
     struct MSFListCellView: View {
         var cell: MSFListVnextCellData
         var layoutType: MSFListCellVnextLayoutType
+        @State private var isExpanded: Bool = false
```

Comment:
Is the calling code able to configure the initial value of a cell's sub items?
For example, the Left Nav wants to have all the users listed when it loads, but still with the capability of collapsing them if the header is tapped.